extensions/machine-learning-services/src/common/queryRunner.ts

```diff
@@ -86,7 +86,21 @@ export class QueryRunner {
 	 * Returns true if python installed in the give SQL server instance
 	 */
 	public async isPythonInstalled(connection: azdata.connection.ConnectionProfile): Promise<boolean> {
-		let result = await this.runQuery(connection, checkPythonInstalledQuery);
+		return this.isLanguageInstalled(connection, 'Python');
```

Comment:
>'Python' [](start = 46, length = 8)

It would be good not to have hard-coded values like these scattered about - I'd suggest adding them to a single constants file instead